contributors/contributors/contribution-guidelines.md

```diff
@@ -81,19 +82,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
 
 ### 1. Correction
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
+**Community Impact:** Use of inappropriate language or other behavior deemed
```

Comment:
Should we specify "inappropriate" or have a link to somewhere that does specify it?